Can I get my date picker into the mainline code?

emilygimm opened the pull request.

emilygimm committed:
Generated date component

emilygimm committed:
date component works

emilygimm committed:
Update pizza-toppings.component.html

emilygimm committed:
Update pizza-toppings.component.ts

emilygimm committed:
Update app.module.ts

emilygimm committed:
added module imports

emilygimm committed:
finished date picker component slack-n-tell

emilygimm committed:
added cancel button

emilygimm committed:
Merge branch 'master' into egimm/week-7-slack-n-tell

emilygimm committed:
fixed duplicate imports

tsteelematc reviewed (commented):
Can you move your component to the bottom?

tsteelematc commented on src/app/pizza-toppings/pizza-toppings.component.html:
You shouldn't need any changes in this file.

emilygimm replied:
I was getting an error with this on line 29. 

 (change)="calculateTotal()"

tsteelematc commented on src/app/pizza-toppings/pizza-toppings.component.ts:
You shouldn't need any changes in this file.

emilygimm replied:
I was getting an error for having multiple properties with the same name on this page specifically checkAll and uncheckAll

emilygimm replied:
`  checkAll = () => this.availablePizzaToppings = this.availablePizzaToppings.map(
    x => ({
      ...x
      ,checked: true
      ,checked: true
    })
  );`

tsteelematc replied:
Right. It looks like those are merge conflicts. I don't think there are dupe props in master version.

emilygimm committed:
moved date component to bottom of page

tsteelematc committed:
Helping craft a cleaner PR...

Making these 3 files match their version in `master` since they don't need to change...
